src/locales/vocabulary.js

```diff
@@ -99,47 +100,42 @@ export default {
   'editdialog.anonymousMessage2_login': 'log in to OpenStreetMap',
   'editdialog.anonymousMessage3': ', your changes will be immediate.',
   'editdialog.add_major_tag': 'Add',
-  'editdialog.location_checkbox': 'Change location',
+  'editdialog.location_checkbox': 'New location',
   'editdialog.location_placeholder': 'eg. across the street',
   'editdialog.location_editor_to_be_added': 'The position cannot be edited here yet, you can do so in the <a href="__link__">iD editor</a>.',
-  'editdialog.place_cancelled': 'Place permanently closed (delete)',
+  'editdialog.place_cancelled': 'Permanently closed (delete)',
   'editdialog.comment': 'Comment (optional)',
-  'editdialog.comment_placeholder': 'link to the source of information etc.',
+  'editdialog.comment_placeholder': 'Extra details, link to source, etc.',
   'editdialog.info_edit': `Your edit will be immediately saved to the OpenStreetMap. Please,
          enter only information from your own or verified sources. It is prohibited
          to copy copyrighted data (e.g. Google Maps). <a href="https://wiki.openstreetmap.org/wiki/How_We_Map">More info</a>`,
-  'editdialog.info_note': `Your suggestion will be processed by OpenStreetMap volunteers. Here
-         you can add an additional note or describe position adjustment etc.
-         It is also appropriate to support your contribution with a link to a source of information (web,
-         photo, etc.).`,
-  'editdialog.other_tags': 'Other properties – tags',
+  'editdialog.info_note': `Your suggestion will be reviewed by editors. You can add additional information for them below! If possible, please provide a link to the source that your suggestion is based on.`,
+  'editdialog.other_tags': 'Edit Tags',
```

Comment:
This text works both as a button label and heading. I think the `Other properties – tags` worked better. What was your issue with it? Maybe again split into two?

Reply:
Went this route because it's more direct and accurate about what you're actually doing.

"Other properties - Tags" is a bit ambiguous and you would have to click the button to actually see what was inside.

Maybe:
Button: "Edit tags"
Heading: "Other tags"

Reply:
Yes, lets split it if it makes sense to you. I was also not sure how "tags" sounds to a native speaker. Is it clear we talk about feature properties?

Reply:
Tags is the word used throughout OSM for the key:value pairs in English. I've seen some discussion that referring to them as "properties" is too generic as well. (Everything about an object can be seen as a "property" of it, tags refer to the specific key:value pairs.